server/src/api/comment.rs

```diff
@@ -82,9 +82,10 @@ impl Perform for Oper<CreateComment> {
 
     // Check for a community ban
     let post = Post::read(&conn, data.post_id)?;
-    if CommunityUserBanView::get(&conn, user_id, post.community_id).is_ok() {
-      return Err(APIError::err("community_ban").into());
-    }
+    let _ban_check = match CommunityUserBanView::get(&conn, user_id, post.community_id) {
+        Ok(_) => { return Err(APIError::err("community_ban").into()) },
+        Err(_e) => {},
```

Comment:
These are unnecessary. In general, don't use match or if let, unless you're going to use the value that it unwraps.